assets/app/scripts/controllers/catalog/images.js

```diff
@@ -14,6 +14,8 @@ angular.module('openshiftConsole')
     $scope.builders = [];
     $scope.nonBuilders = [];
     $scope.sourceURL = $routeParams.builderfor;
+    $scope.loadedProjectImageStreams = false;
+    $scope.loadedSharedImageStreams = false;
```

Comment:
Don't make the view check both of these... just have it check a `loaded` flag (or something similar), and set that to `true` once both are loaded.